package.json

```diff
@@ -9,16 +9,18 @@
     "ci:publish": "lerna publish from-package --yes",
     "ci:prepublish": "lerna publish from-package --no-git-reset --yes --dist-tag next",
     "release": "lerna version --no-git-tag-version",
-    "verify": "turbo run lint && turbo run typescript && yarn run test",
-    "prettier": "prettier --config prettier.config.js --write \"packages/**/*.{ts,tsx}\"",
+    "verify": "turbo run lint && turbo run typescript",
+    "prettier": "prettier --config prettier.config.js --write \"packages/**/*.{ts,tsx,css,scss}\" \"e2e/**/*.ts\"",
     "postinstall": "husky install",
-    "test": "cross-env TZ=UTC jest --config jest.config.js --verbose false --passWithNoTests",
-    "test-watch": "cross-env TZ=UTC jest --watch --config jest.config.js",
+    "test": "cross-env TZ=UTC jest --config jest.config.json --verbose false --passWithNoTests",
+    "test-watch": "cross-env TZ=UTC jest --watch --config jest.config.json",
     "coverage": "yarn test --coverage",
-    "ci:bump-form-engine-lib": "yarn up @openmrs/openmrs-form-engine-lib@next"
+    "ci:bump-form-engine-lib": "yarn up @openmrs/openmrs-form-engine-lib@next",
```

Comment:
You don't have to replace the content of this file. adding `test-e2e` command and the dependencies is sufficient.